docs/docs/examples/elasticsearch/BITBUCKET_ELASTICSEARCH.md

```diff
@@ -12,28 +12,40 @@ helm repo add elastic https://helm.elastic.co
 ```
 then install it:
 ```bash
-helm install elasticsearch --version 7.9.3 elastic/elasticsearch
+helm install elasticsearch --set image.tag="7.9.3" elastic/elasticsearch
 ```
+!!! info "Pre-requisites of Elasticsearch Helm chart"
+    Running the above commands will install Elasticsearch with default configuration, i.e. 3 worker nodes. 
+    However, it may not always work out of the box if failed to fulfill pre-requisites for the default installation. 
+    Some example pre-requisites include:
+
+    * CPU/memory requests: 1000m/2Gi
```

Comment:
I think CPU and Memory is for each worker. As the storage this is mentioned, is good to mention `for each worker` here to make it consistent.